x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -343,7 +343,7 @@ public synchronized OperationMode getOperationMode() {
     }
 
     /** Return true if the license is currently within its time boundaries, false otherwise. */
-    public synchronized boolean isActive() {
+    public synchronized boolean allowForAllLicenses() {
```

Comment:
Address feedback https://github.com/elastic/elasticsearch/pull/51864#discussion_r376829172

Reply:
I think the javadoc should be updated. Stricly it's accurate, but by intent this method now has a different purpose.

Reply:
I am having some difficulty trying to rephrase this without the word "active" or something that explains it. How about "Return true if the feature is allowed by all active licenses"?